.eslintrc

```diff
@@ -0,0 +1,189 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "google"
+  ],
+  "rules": {
+    "no-empty": [
+      2,
```

Comment:
The `2` actually stems from [here](http://eslint.org/docs/user-guide/configuring#configuring-rules). To make it more readable, I converted it to the text form (`"error"`)